extract_unique_sensors_01.py

```diff
@@ -1,6 +1,14 @@
+import argparse
 from urllib.request import urlopen
 from pathlib import Path
 
+parser = argparse.ArgumentParser(description='Extract unique sensors')
+parser.add_argument('--file_name', help='Actual name of the file')
+parser.add_argument('--url', help='Location of SensorList.txt')
+parser.add_argument('--kat_sensor', help='Name of unique sensor')
+
+args = parser.parse_args()
+
 file_name = "SensorList.txt"
 url = "http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt"                                                                          
 kat_sensor = "kat.sensors"
```

Comment:
Why do you need to override the args you're passing in your script?

Reply:
Please pay attention to detail, this is crucial. 
Every code commit should be reviewed by the author preferably before others review it. 

Always, think about: "What could go wrong?"